crates/assistant/src/assistant_panel.rs

```diff
@@ -911,6 +924,15 @@ impl AssistantPanel {
             });
 
             self.show_context(editor.clone(), cx);
+            let workspace = self.workspace.clone();
+            cx.spawn(move |_, mut cx| async move {
+                workspace
+                    .update(&mut cx, |workspace, cx| {
+                        workspace.focus_panel::<AssistantPanel>(cx);
+                    })
+                    .ok();
+            })
+            .detach();
```

Comment:
This allows us to: 
1. keep the current new tab behavior (new tab is to the immediate right of the current open context)
2. move the cursor focus to the new context